extensions/wikia/SpecialVideos/SpecialVideosHelper.class.php

```diff
@@ -149,7 +97,7 @@ public function getVideos( $sort, $page, $providers = [], $category = '', $optio
 			$videoDetail = $helper->getVideoDetail( $videoInfo, $videoOptions );
 			if ( !empty( $videoDetail ) ) {
 				$byUserMsg = WikiaFileHelper::getByUserMsg( $videoDetail['userName'], $videoDetail['timestamp'] );
-				// FIXME: this isn't shown to users b/c it's buggy (CONSF-51)
+				// SUS-78 | Not used in template but used by API clients - GameGuides App
 				$viewTotal = wfMessage( 'videohandler-video-views', $this->wg->Lang->formatNum( $videoDetail['viewsTotal'] ) )->text();
```

Comment:
Isn't that connected to https://github.com/Wikia/app/pull/9673/files#diff-e10911b408f376068e7330dad13f1fd2L548?